### PR TITLE
Add fixed overhead to external grader timeout

### PR DESCRIPTION
**Pull request by nwalters512**
When instructors specified a really short timeout for their external grading jobs, our naive approach of computing an overall timeout by multiplying the question timeout by 2 fell apart, as that couldn't account for the overhead of pulling the image and so on. So, we'll instead just add a generous amount of time (default 5 minutes) to the instructor timeout to account for this.

Closes https://github.com/PrairieLearnInc/sysconf/issues/781.

### CODE REVIEW

File: grader_host/config/base.yaml [hunk 1]
```diff
@@ -74,9 +74,14 @@ properties:
   resultsQueueUrl:
     default: null
     envVar: RESULTS_QUEUE_URL
+
+  # Timeouts
```
Comment (nwalters512) [inline grader_host/config/base.yaml:78]: Man, who introduced this bespoke config format 😉

Comment (mwest1066) [reply grader_host/config/base.yaml:78]: 🤣

File: grader_host/index.js [hunk 2]
```diff
@@ -406,8 +411,8 @@ function runJob(info, callback) {
   const globalJobTimeoutId = setTimeout(() => {
```
Comment (mwest1066) [inline grader_host/index.js:411]: Rename this variable to `jobTimeoutId` for consistency with the fact that we aren't using `globalJobTimeout` anymore?

**Review by mwest1066 — APPROVED** [review]

File: grader_host/index.js (hunk 2 above)
Comment (nwalters512) [reply grader_host/index.js]: Done in 0ec0f34.

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**